### PR TITLE
Add clj-kondo-lsp

### PR DESCRIPTION
**Pull request by Baspar**
Adding a new LSP for Clojure: https://github.com/borkdude/clj-kondo.lsp

### CODE REVIEW

File: installer/install-clj-kondo-lsp.cmd [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@echo off
+
+curl -L -o clj-kondo-lsp https://github.com/borkdude/clj-kondo/releases/download/v2020.05.09/clj-kondo-lsp-server-2020.05.09-standalone.jar
```
Comment (mattn) [inline installer/install-clj-kondo-lsp.cmd:3]: Could you please let new variable `version=2020.05.09` ?

File: README.md [hunk 2]
```diff
@@ -90,6 +90,7 @@ Because there is no way to update a server, please run `:LspInstallServer` again
 | C/C++            | clangd                                                 | Yes           |
 | C#               | omnisharp                                              | Yes           |
 | Clojure          | clojure-lsp                                            | Yes           |
+| Clojure          | clj-kond-lsp                                           | Yes           |
```
Comment (mattn) [inline README.md:93]: s/kond/kondo/ ?

File: installer/install-clj-kondo-lsp.cmd (hunk 1 above)
Comment (Baspar) [reply installer/install-clj-kondo-lsp.cmd]: Fixed in 1941db0

File: README.md (hunk 2 above)
Comment (Baspar) [reply README.md]: Fixed in 1941db0

**Comment by Baspar** [comment]
@mattn Fixed both comment

File: installer/install-clj-kondo-lsp.cmd [hunk 3]
```diff
@@ -1,3 +1,3 @@
 @echo off
-
-curl -L -o clj-kondo-lsp https://github.com/borkdude/clj-kondo/releases/download/v2020.05.09/clj-kondo-lsp-server-2020.05.09-standalone.jar
+New-Variable -Name "VERSION" -Value "2020.05.09"
```
Comment (mattn) [inline installer/install-clj-kondo-lsp.cmd:2]:
```suggestion
setlocal
set VERSION=2020.05.09
```

File: installer/install-clj-kondo-lsp.cmd [hunk 4]
```diff
@@ -1,3 +1,3 @@
...
+curl -L -o clj-kondo-lsp https://github.com/borkdude/clj-kondo/releases/download/v$VERSION/clj-kondo-lsp-server-$VerSION-standalone.jar
```
Comment (mattn) [inline installer/install-clj-kondo-lsp.cmd:3]:
```suggestion
curl -L -o clj-kondo-lsp https://github.com/borkdude/clj-kondo/releases/download/v%VERSION%/clj-kondo-lsp-server-%VERSION%-standalone.jar
```

Comment (Baspar) [reply installer/install-clj-kondo-lsp.cmd:4]: I'm good with that, not a pro on powershell ><

**Comment by mattn** [comment]
Thank you

**Pull request merged**